src/SWP/Bundle/CoreBundle/Service/FacebookInstantArticlesService.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Service;
+
+use Facebook\InstantArticles\Client\Client;
+use Facebook\InstantArticles\Client\InstantArticleStatus;
+use Facebook\InstantArticles\Elements\InstantArticle;
+use SWP\Bundle\ContentBundle\Model\ArticleInterface;
+use SWP\Bundle\CoreBundle\Model\FacebookInstantArticlesArticle;
+use SWP\Bundle\CoreBundle\Model\FacebookInstantArticlesFeedInterface;
+use SWP\Bundle\FacebookInstantArticlesBundle\Manager\FacebookInstantArticlesManagerInterface;
+use SWP\Bundle\StorageBundle\Doctrine\ORM\EntityRepository;
+use SWP\Component\Storage\Factory\FactoryInterface;
+
+class FacebookInstantArticlesService
+{
+    /**
+     * @var FacebookInstantArticlesManagerInterface
+     */
+    protected $facebookInstantArticlesManager;
+
+    /**
+     * @var FactoryInterface
+     */
+    protected $instantArticlesArticleFactory;
+
+    /**
+     * @var EntityRepository
+     */
+    protected $facebookInstantArticlesArticleRepository;
+
+    /**
+     * FacebookInstantArticlesService constructor.
+     *
+     * @param FacebookInstantArticlesManagerInterface $facebookInstantArticlesManager
+     * @param FactoryInterface                        $instantArticlesArticleFactory
+     * @param EntityRepository                        $facebookInstantArticlesArticleRepository
+     */
+    public function __construct(
+        FacebookInstantArticlesManagerInterface $facebookInstantArticlesManager,
+        FactoryInterface $instantArticlesArticleFactory,
+        EntityRepository $facebookInstantArticlesArticleRepository
+    ) {
+        $this->facebookInstantArticlesManager = $facebookInstantArticlesManager;
+        $this->instantArticlesArticleFactory = $instantArticlesArticleFactory;
+        $this->facebookInstantArticlesArticleRepository = $facebookInstantArticlesArticleRepository;
```

Comment:
`fbiaArticleRepository` would be better as its shorter and the meaning is the same